scripts/publish/src/getRegistryVersion.js

```diff
@@ -2,6 +2,15 @@ const { exec } = require('shelljs');
 
 // Get version of package in NPM regsitry. Returns 0.0.0 if doesn't exist.
 module.exports = name => {
-  const npmVersion = exec(`npm show ${name} version`, { silent: true }).stdout;
-  return npmVersion ? npmVersion.split('\n')[0] : '0.0.0';
+  let packageVersions = [];
+  const npmVersion = exec(`npm view ${name} versions -json`, { silent: true })
```

Comment:
Sorry to be a buzzkill but this creates a serious command injection vulnerability. May I suggest using [spawn](https://nodejs.org/api/child_process.html). Using `spawn` enforces the command and parameter separation, that way you should remove this potential command injection.